### PR TITLE
Fixes  #102: Rename model functions to `model_x()`

### LINKED ISSUE #102: Rename model functions to `model_x()`
This issue is to request that all model functions, which are currently named `epidemic_x_language()` should be renamed to `model_x_language()`. For instance, the function `epidemic_default_cpp()` would be `model_default_cpp()`.

### PR DESCRIPTION
**Pull request by bahadzie**

### CODE REVIEW

**Comment by pratikunterwegs** [comment]
Thanks @bahadzie - will take a look later today.

File: inst/WORDLIST [hunk 1]
```diff
@@ -85,6 +85,8 @@ gh
 github
 io
 kylieainslie
+linter
+lintr
```
Comment (pratikunterwegs) [inline inst/WORDLIST:89]: I'm curious where these additions are from

**Review by pratikunterwegs — COMMENTED** [review]
Thanks @bahadzie for this - since it's a change that affects nearly every file in the codebase, I will merge this after #132 and #126 as that seems like the way to minimise merge conflicts - although there will be a lot of those anyway.

Further branches/PRs should probably wait until this set of PRs is merged, to avoid more merge conflicts still.

File: inst/WORDLIST (hunk 1 above)
Comment (bahadzie) [reply inst/WORDLIST:89]: I added them to fix some error message when the GH Actions ran. Not sure why they were flagged though.

**Pull request merged**